stable/tomcat/templates/appsrv.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "tomcat.fullname" . }}
+  labels:
+    app: {{ template "tomcat.fullname" . }}
```

Comment:
Please follow the principles when defining labels.  Please refer https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#dependencies.
```
name: {{ template "myapp.fullname" . }}
labels:
  app: {{ template "myapp.name" . }}
  chart: {{ template "myapp.chart" . }}
  release: {{ .Release.Name }}
  heritage: {{ .Release.Service }}
```

Reply:
I used `tomcat.fullname` and `tomcat.chart`. I was not sure if the guidelines require `myapp` instead of `tomcat`.

Reply:
I just mean app should be `template "tomcat.name" .`

Reply:
ok makes sense. fixed